### DISCUSSION TITLE
packetloss.sh gives an error every 10 min though ZEEK is disabled

### DISCUSSION THREAD

**Discussion by Masaya-A** [Security-Onion-Solutions/securityonion#5746] [Unsupported Versions] [answered]
Security Onion version: 2.3.80
Install source: 2.3.30 by Network
OS: CentOS 7
Install type: Standalone

I have added following to `minions/$(uname -n)_standalone.sls` after updated from 2.3.70 to 2.3.80 and did highstate.
```
zeek:
  enabled: False
```

ZEEK is now correctly disabled and `so-status` shows so-zeek is DISABLED.
However, `zeekpacketlosscron` in the `zeek/init.sls` still adds a cron job `packetloss.sh`, so I get following error from cron daemon every 10 mins.
`Error: No such container: so-zeek`

Could you fix this, please?

**Comment by m0duspwnens** [comment, marked as answer]
Hi @Masaya-A, thanks for the report. We have opened [issue #5759](https://github.com/Security-Onion-Solutions/securityonion/issues/5759) to track this.
